supervisord.conf

```diff
@@ -0,0 +1,26 @@
+[supervisord]
+nodaemon=true
+
+[program:xl2tpd]
+command=/usr/sbin/xl2tpd -c /etc/xl2tpd/xl2tpd.conf -D
+redirect_stderr=true
+numprocs=1
+stdout_logfile=/dev/fd/1
+stdout_logfile_maxbytes=0
+
+[program:ipsec]
```

Comment:
Maybe it's worth having `autorestart=true` here? Or do you think when one of the processes crashes it's better to rely on docker's `--restart always` to cycle the entire container?